### ISSUE TITLE
Customization example

### ISSUE THREAD

**Issue by Seybo** [narajaon/onestatus#5]
Hi. And thank you for your plugin. It's really nice and way less popular than i would expect (but I'm pretty sure it's just because it's new)

I'm trying to understand how to customize it and i think my lacking of vimscript knowledge doesn't allow me to do it.
Here is what i have now (using tmuxinator):
![image](https://user-images.githubusercontent.com/10584167/92325533-549c8e80-f069-11ea-9990-6fba8c00e9d8.png)

And what i would like to achieve is these three things:
1) remove hostname `carrot` from the left
2) move all the tabs names from the middle to the left
3) change the colors of the right to match the colors on the left

I'm not sure if 1 and 2 should be done through your plugin's config or through the tmux's config.

**Comment by tusqasi** [comment]
problem 1.
For now you can change
`onestatus/plugin/onestatus.vim`
line 51 *label*
`let s:left = { -> {'command': 'set-option -g status-left', 'attributes': [{"fg": "#6c757d", "bg": "default", "label": "{change this to what ever you want}"}]}}`

for example. Here it's just a space
`let s:left = { -> {'command': 'set-option -g status-left', 'attributes': [{"fg": "#6c757d", "bg": "default", "label": " "}]}}`

**Comment by tusqasi** [comment]
problem 2.
`onestatus/plugin/onestatus.vim`
line 63 *center to left*
the default file
 ``` 
62 call onestatus#build([
63       \{'command' : 'set-option -g status-justify center'},
64       \{'command': 'set-option status-right-length 50'},
65       \{'command': 'set-option status-left-length 60'},
67       \])
```
make center left
 ``` 
62 call onestatus#build([
63       \{'command' : 'set-option -g status-justify left'},
64       \{'command': 'set-option status-right-length 50'},
65       \{'command': 'set-option status-left-length 60'},
67       \])
```

**Comment by tusqasi** [comment]
problem 3. 
This should be done through changing variable values which is not how it is implemented currently
It will be unnecessarily painful to do manually.
But you can change them through editing the file itself.

tl;dr 
wait till it is implemented or someone forks and does the variable shenanigans

**Comment by narajaon** [comment]
Hi, thanks for using onestatus ! It's not that popular also because I plan to make the API much simpler before advertising it more.
Look forward to it this weekend.
And also thanks @tusqasi for helping out

**Comment by narajaon** [comment]
@Seybo The plugin as been updated to v0.2 and there's a fresh README that you can follow to customize your onestatus !
Any feedback would be appreciated. Thanks !
